src/components/footer/_macro.njk

```diff
@@ -38,8 +38,6 @@
                     "altTextId": 'poweredby-alt'
                 })
             }}
-        {% else %}
```

Comment:
Why is this part being removed? it doesn't seem to be related to the `params.poweredBy.partnership` thing which is being raised.

Reply:
If you follow the logic whatever logo is being set here is being used in the `params.poweredBy.partnership` bit below. To keep the same functionality I had to in effect move this else down there